### ISSUE TITLE
Switch from MMdetection to plain PyTorch

### ISSUE THREAD

**Issue by mzur** [biigle/maia#185]
After the [migration](https://github.com/biigle/maia/pull/117) from the TensorFlow-based Mask R-CNN to MMDetection because the model code was no longer maintained, we now face a similar situation with MMDetection which is not maintained any more. We should switch to a [plain PyTorch model of Faster R-CNN](https://pytorch.org/vision/main/models.html#object-detection) before there are security vulnerabilities that we can no longer patch. A plain PyTorch model will hopefully work for a while longer. Maybe we could also compare the available object detection models and choose a newer architecture if it performs better.
